ProjectSidewalk/SidewalkWebpage
# Adds unexplored tasks to minimap

Resolves #3170

On page load, render all tasks (only time this will be done). Every time task is completed, update only the new current task (render: erase gray and draw red/green) once old current task has ended. If user jumps to another location within the same neighborhood, store old current task (incomplete) right before new current task is set. Then once new current task is set, rerender the old current task.

##### Before/After screenshots (if applicable)
![3170_minimap_tasks_fix](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/106000281/123bac3c-878d-4754-a480-2b514497d068)


##### Testing instructions
1. Navigate to Explore page
2. See minimap
3. Jump within neighborhood by selecting Jump in left side bar, then 'I want to explore another area!' then 'Go to a new location in this neighborhood'

##### Things to check before submitting the PR
